waifus/waifu.js

```diff
@@ -0,0 +1,35 @@
+/*Adds Waifu to custom New Tab Page*/
```

Comment:
Does the `waifu/` directory need to be outside of the firefox directory?

Reply:
I'm confused  with this particular comment. Can you clarify what you mean?

Reply:
Please see this comment https://github.com/doki-theme/doki-theme-chrome/pull/13#discussion_r546228041